dist/assets/css/donation-banner.css

```diff
@@ -0,0 +1,7 @@
+.donation-bnanner__icon {
+  width: 200px;
+  height: 200px;
+  top: -100px;
+  left: 50%;
+  transform: translateX(-50%);
+}
```

Comment:
Perfect element centering 👍 
Another you can try also is to use the `calc()` function.  [MDN docs link](https://developer.mozilla.org/en-US/docs/Web/CSS/calc).

**Check the attached photo** 

![circle](https://user-images.githubusercontent.com/8650435/79672573-5feb3c80-81e4-11ea-99f9-f7f14d7cf3bd.PNG)

Reply:
Honestly, I liked your comment, and I will try your suggestion, thank you it was a helpful tip

Reply:
you did a great job. Keep going